tgui/packages/tgui/http.ts

```diff
@@ -0,0 +1,19 @@
+import { FetchOptions } from "jsdom";
```

Comment:
Shouldn't import this type from jsdom, which is primarily used for testing. I think the real type is called `RequestInit`.